SU2_CFD/include/output/filewriter/CParallelDataSorter.hpp

```diff
@@ -31,6 +31,7 @@
 #include "../../../../Common/include/option_structure.hpp"
 #include "../../../../Common/include/toolboxes/CLinearPartitioner.hpp"
 
+
```

Comment:
extra line?

Reply:
I somehow like 2 line spacing between some parts in a file so that it is better noticeable while scrolling e.g. in between function implementations in .cpp files. But I removed that newline now ... I guess its a matter of preference